processing/src/main/java/org/apache/druid/query/aggregation/mean/DoubleMeanAggregatorFactory.java

```diff
@@ -106,8 +108,12 @@ public VectorAggregator factorizeVector(final VectorColumnSelectorFactory select
   }
 
   @Override
-  public boolean canVectorize()
+  public boolean canVectorize(ColumnCapabilitiesProvider columnCapabilitiesProvider)
   {
+    if (fieldName != null) {
```

Comment:
fieldName cannot be null for this one, there is a precondition in the constructor

Reply:
Done